README.md

```diff
@@ -20,56 +20,99 @@
     <img alt="Build" src="https://img.shields.io/github/workflow/status/p-yukusai/PyWall/PyWall%20CI?style=for-the-badge">
 </a>
 </h1>
-  
+
+# PyWall
+
+PyWall is a Python-based firewall application. This tool helps manage network traffic by creating and configuring firewall rules. Note that PyWall does not monitor network traffic itself.
+
+## Features
+
+- Configure firewall rules
+- Block or allow connections based on IP, port, or application
```

Comment:
Not true, we simply generate or delete a firewall rule for a specific file, we can generate different kinds of rules but they don't block based on IP or port but instead they block inbound or outbound connections per file using Window's firewall.

Reply:
Done